src/helpers/url.js

```diff
@@ -45,6 +45,7 @@ const getWopiUrl = ({ fileId, title, readOnly, closeButton, revisionHistory, tar
 		+ (revisionHistory ? '&revisionhistory=1' : '')
 		+ (readOnly ? '&permission=readonly' : '')
 		+ (target ? '&target=' + encodeURIComponent(target) : '')
+		+ (Config.get('startPresentation') ? '&startPresentation=1' : '')
```

Comment:
I think we should read into a variable once at page load and make sure to reset it to false if it was used once

Reply:
Once the page is refreshed (which, in its current state, it has to be) then the initial state will reset and `startPresentation` will not be enabled unless you again append it to the URL. I'm assuming once I implement a way to close the presentation mode this way, then it would be relevant?